scripts/frontmatter_tagging/frontmatter_tagging.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+
+# This script iterates over our docs and converts version front matter to tags format
+# sectionTags are not dealt with in this script
+import os
+import glob
+import sys
+from pathlib import Path
+import re
+
+## Select desired markdown and adoc files
+docs_paths = (
+  '../../jekyll/_cci2/*.md',
+  '../../jekyll/_cci2/*.adoc',
+  '../../jekyll/_cci2_ja/*.md',
+  '../../jekyll/_cci2_ja/*.adoc',
```

Comment:
Can this be removed since Mayuko requested to no have the script impact the content?